### ISSUE TITLE
postgresql_db_user_pass is undefined

### ISSUE THREAD

**Issue by ghost** [palkan-ansible/postgresql#1]
```
  - hosts: servers
    roles:
       - palkan.postgresql
```

This basic playbook triggers the message below:

```
FAILED! => {"failed": true, "msg": "ERROR! 'postgresql_db_user_pass' is undefined"}
```

**Comment by palkan** [comment]
@olo-dw Thanks for the report!

Fixed.

**Comment by ghost** [comment]
My I suggest you to start your commit message by "Fixes #\<issue number\>" in order to automatically close the issue.

Source: https://help.github.com/articles/closing-issues-via-commit-messages/

Fixed by commit c0dbeff95c6608b231a47760c1cf61fee0ac5854.
